pom.xml

```diff
@@ -515,6 +515,17 @@
       </build>
     </profile>
     <profile>
+      <id>spark-2.2</id>
+      <properties>
+        <spark.version>2.1.0</spark.version>
+        <scala.binary.version>2.11</scala.binary.version>
+        <scala.version>2.11.8</scala.version>
```

Comment:
Should not hard code the version

Reply:
this is not hard code. by default use spark 2.1.0, here should change to 2.2.0